lib/BenchShow/Report.hs

```diff
@@ -21,12 +22,52 @@ import Control.Applicative (ZipList(..))
 import Control.Monad (forM_)
 import Data.Maybe (fromMaybe)
 import Statistics.Types (Estimate(..))
+
+#ifdef WL_PRETTY
 import Text.PrettyPrint.ANSI.Leijen hiding ((<$>))
+#endif
+
 import Text.Printf (printf)
 
 import BenchShow.Common
 import BenchShow.Analysis
 
+#ifndef WL_PRETTY
+
+--------------------------------------------------------------------------------
+-- A replacement for all the combinators used from  Text.PrettyPrint.ANSI.Leijen
+--------------------------------------------------------------------------------
+
+type Doc = String
+
+dullred :: Doc -> Doc
+dullred = id
+
+dullgreen :: Doc -> Doc
+dullgreen = id
+
+(<+>) :: Doc -> Doc -> Doc
+(<+>) x y = x ++ " " ++ y
+
+vcat :: [Doc] -> Doc
+vcat = unlines
+
+putDoc :: Doc -> IO ()
+putDoc = putStrLn
+
+fill :: Int -> Doc  -> Doc
+fill i x =
+    let len = length x
+     in x ++ replicate (i - len) ' '
+
+indent :: Int -> Doc -> Doc
+indent i = unlines . map (replicate i ' ' ++) . lines
+
+text :: String -> Doc
+text = id
+
+#endif
+
```

Comment:
We can put this in a separate module abstracting the pretty functionality, and include that module here.